Update Dockerfile

lvthillo opened the pull request:
Update 17.06-ce

lvthillo committed:
Update Dockerfile

Update 17.06-ce

radu-matei commented:
Why close this?
I would be interested in merging this (provided we only keep the Docker client - in my repo I install both the client and the engine - I would only want to keep the client, since we are using the engine from the host)

lvthillo commented:
@radu-matei I closed this because it forgot to mention this part to:
`&& curl -fSL "https://${DOCKER_BUCKET}/builds/Linux/x86_64/docker-${DOCKER_VERSION}.tgz" -o docker.tgz \`
Is now
`&& curl -fSL "https://${DOCKER_BUCKET}/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz" -o docker.tgz \`

lvthillo committed:
Update Dockerfile

lvthillo commented:
Before I can test further, I think they need to update the jenkins slave alpine image to solve the curl issue: ```
+ curl -V
curl 7.55.0 (x86_64-alpine-linux-musl) libcurl/7.54.0
```
libcurl is an older version than curl.

lvthillo committed:
Update Dockerfile

lvthillo committed:
Update Dockerfile

add curl-dev to solve curl issue: curl: (48) An unknown option was passed in to libcurl

lvthillo commented:
Do you want to client only? Check the last comments here: https://github.com/moby/moby/issues/15360 : 

> While there's no separate package for just the client, I suggest to download a .tgz with static binaries (including the client);
